tests/event-log/event-log-level.spec.ts

```diff
@@ -0,0 +1,130 @@
+import chaiAsPromised from 'chai-as-promised';
+import { computeCid } from '../../src/utils/cid.js';
+import { EventLogLevel } from '../../src/event-log/event-log-level.js';
+import { TestDataGenerator } from '../utils/test-data-generator.js';
+import chai, { expect } from 'chai';
+
+chai.use(chaiAsPromised);
+
+let eventLog: EventLogLevel;
+
+describe('EventLogLevel Tests', () => {
+  describe('append tests', () => {
+    before(async () => {
+      eventLog = new EventLogLevel({ location: 'TEST-EVENTLOG' });
+      await eventLog.open();
+    });
+
+    beforeEach(async () => {
+      await eventLog.clear();
+    });
+
+    after(async () => {
+      await eventLog.close();
+    });
+
+    it('appends a tenant namespaced entry into leveldb', async () => {
+      const { requester, message } = await TestDataGenerator.generateRecordsWrite();
+      const messageCid = await computeCid(message);
+
+      const watermark = await eventLog.append(requester.did, messageCid.toString());
+
+      for await (const [key, value] of eventLog.db.iterator()) {
+        expect(key).to.include(watermark);
+        expect(value).to.equal(messageCid.toString());
+      }
```

Comment:
The use of loop confused me... consider:
```suggestion
      const iteratorResult = await eventLog.db.iterator().next();
      const [key, value] = iteratorResult.value;
      expect(key).to.include(watermark);
      expect(value).to.equal(messageCid);
```

Reply:
the lack of loop confuses me 😅 . Happy to use whatever most people find least confusing. @dcrousso @diehuxx thoughts?

Reply:
perhaps we do a bit of both?
```
const entries = await asyncGeneratorToArray(eventLog.db.iterator());
expect(entries.length).to.equal(1);
expect(entries[0][0]).to.include(watermark);
expect(entries[0][1]).to.equal(messageCid);
```

Reply:
@thehenrytsai, i'm going to use the fact that `for await (const [key, value] of eventLog.db.iterator()) {` [already exists in other parts of the codebase](https://github.com/TBD54566975/dwn-sdk-js/blob/main/src/store/level-wrapper.ts#L224-L226) (aka the consistency argument) to my benefit

![](https://media.giphy.com/media/l0MYDaZrMxVz4p0pq/giphy.gif)

Reply:
No worries, not something worth arguing over!